File: Slowest Key/Sk.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution
{
public:
    char slowestKey(vector<int> &rt, string keypress)
    {
        int maxTime = rt[0];
        char result = keypress[0];

        for (int i = 1; i < rt.size(); i++)
        {
            int current = rt[i] - rt[i - 1];
            if (current > maxTime || current == maxTime && keypress[i] > result)
            {
                maxTime = current;
                result = keypress[i];
            }
        }

        return result;
    }
};

File: Slowest Key/Sk.java
public class Sk{
    public char slowestKey(int[] rt, String keypress) {
        int maxTime = rt[0];
        char result = keypress.charAt(0);

        for (int i = 1; i < rt.length; i++) {
            int current = rt[i] - rt[i - 1];
            if (current > maxTime || current == maxTime && keypress.charAt(i) > result) {
                maxTime = current;
                result = keypress.charAt(i);
            }
        }

        return result;
    }
}